src/doc/book/structs.md

```diff
@@ -61,7 +61,7 @@ write something like this:
 
 ```rust,ignore
 struct Point {
-    mut x: i32,
+    mut x: i32, // this causes an error
```

Comment:
Would like to confirm as it will affect a bunch of files which are already fairly consistent in their current  style.
Here are some examples:

benchmark-tests.md:113
    -    `// note lack of `;` (could also use an explicit `return`)`
    +    `// Note lack of `;` (could also use an explicit `return`).`

casting-between-types.md:109
    -     `let a = 300 as *const char; // a pointer to location 300`
    +    `let a = 300 as *const char; // A pointer to location 300.`
OR as a sentence with predicate:
    +    `let a = 300 as *const char; // 'a' is a pointer to location 300.`

traits.md:422
    -     `// can be called with T == i32`
    +     `// Can be called with T == i32.`

traits.md:479
    -     `true // overrides the expected value of is_invalid()`
    +     `true // Overrides the expected value of is_invalid().`

choosing-your-guarantees.md:293-296
`let guard = mutex.lock();`
    -       `// guard dereferences mutably to the inner type`
    +       `// 'guard' dereferences mutably to the inner type.`

There's only an empty stub here on the topic: http://aturon.github.io/style/comments.html#code-snippets
Nothing on the topic here: https://github.com/rust-lang-nursery/fmt-rfcs/blob/master/guide/guide.md